src/envariables.c

```diff
@@ -87,9 +87,9 @@ unsigned long INTERNAL qt_internal_get_env_num(const char   *envariable,
     unsigned long tmp = dflt;
 
     if (dflt != 0) {
-        char dflt_str[10];
+        char dflt_str[21];
 
-        snprintf(dflt_str, 10, "%lu", dflt);
+        snprintf(dflt_str, 21, "%lu", dflt);
```

Comment:
Why 21?

Reply:
The max length of an unsigned long is 20 digits. `snprintf` needs one more character than the max size, so 21.